sensu-plugins-jenkins.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |s|
   s.version                = SensuPluginsJenkins::Version::VER_STRING
 
   s.add_runtime_dependency 'sensu-plugin',        '~> 1.2'
-  s.add_runtime_dependency 'rest-client',         '1.8.0'
+  s.add_runtime_dependency 'rest-client',         '2.0.2'
```

Comment:
should we possibly be a bit looser on version constraint here? We could use pessimistic versioning like this `~> 2.0`